### PR TITLE
Merkle tree add get proof by pos

### PR DESCRIPTION
**Pull request by FranFiuba**

### CODE REVIEW

**Comment by ilitteri** [comment]
Can we add a test for this?

**Review by ilitteri — APPROVED** [review]
LGTM

File: crypto/src/merkle_tree/mod.rs [hunk 1]
```diff
@@ -71,6 +71,26 @@ impl<F: IsField, H: IsCryptoHash<F> + Clone> MerkleTree<F, H> {
         None
     }
 
+    pub fn get_proof_by_pos(&self, pos: usize, value: FieldElement<F>) -> Option<Proof<F, H>> {
```
Comment (pablodeymo) [inline crypto/src/merkle_tree/mod.rs:74]: value should be a reference

**Review by pablodeymo — APPROVED** [review]
LGTM

**Review by jrchatruc — APPROVED** [review]

**Pull request merged**